.changeset/hip-lobsters-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": patch
+---
+
+Improve experience for WARP users
```

Comment:
could you be more explicit that we improved the reliability of the final polling step that waits for the new deployment to become available? thanks